github/repos_contents.go

```diff
@@ -72,7 +73,8 @@ func (r *RepositoryContent) Decode() ([]byte, error) {
 	return o, nil
 }
 
-// GetReadme gets the Readme file for the repository.
+// GetReadme gets the Readme file for the repository, which is stored as base64.
```

Comment:
As I type it up, I think it's great to have a note on RepositoryContent, though I want to give people a heads up as they look at each command. I could go either way, but I err on sharing wherever it hurt as a user.